images/router/f5/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#
+# This is the HAProxy router for OpenShift Origin.
+#
+# The standard name for this image is openshift/origin-f5-router
+#
+FROM openshift/origin-base
+
+ADD bin/openshift /usr/bin/openshift
+RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
+
+ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
```

Comment:
I don't understand the question.  This image is built from `openshift/origin-base` rather than from `openshift/origin-haproxy-router-base` so I don't understand why it would require a change to the haproxy Dockerfile.